go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -2186,8 +2195,31 @@ func (fbo *folderBranchOps) identifyOnce(
 	switch {
 	case ei.Behavior.WarningInsteadOfErrorOnBrokenTracks() &&
 		len(ei.GetTlfBreakAndClose().Breaks) > 0:
+		// In this case, the caller has explicitly requested that we
+		// treat proof failures as warnings, instead of errors.  For
+		// example, the GUI does this and shows a warning banner but
+		// still allows access to the files.  Identifies are
+		// expensive, so in this case we skip future identifies that
+		// also have this behavior for a short time period.
 		fbo.log.CDebugf(ctx,
-			"Identify finished with no error but broken proof warnings")
+			"Identify finished with no error but broken proof warnings; "+
+				"caching result for %d", cacheBrokenProofIdentifiesDuration)
+		fbo.identifyDoneWithWarning = true
+		timer := time.NewTimer(cacheBrokenProofIdentifiesDuration)
+		fbo.goTracked(func() {
```

Comment:
Since now checks are moved inside the !alwaysrun block, it's possible we reach here even when identifyDoneWithWarning==true, then we'd have multiple timers. Looks like the timer needs to be move up so we can stop it before starting a new one?

Reply:
You're talking about a case where both `AlwaysRunIdentify == true` and `WarningInsteadOfErrorOnBrokenTracks == true`?  No matter what we can't stop the timer from the previous call into this function unless we save it a way to cancel it somewhere, which doesn't seem worth it for this case that can't even happen.

Reply:
Or actually I can just check whether it's already set in that case, and avoid making a new timer.